### DISCUSSION TITLE
Hey, how do I send messages through Django Admin?

### DISCUSSION THREAD

**Discussion by LuizDMM** [xtrinch/fcm-django#252] [Q&A] [answered]
The title is a little self-descriptive. I installed this package on a project I have, but it didn't show up in Django Admin. How do I set it up?

**Comment by xtrinch** [comment, marked as answer]
Have you checked out the example project? https://github.com/xtrinch/fcm-django-web-demo/blob/master/mysite/mysite/settings.py

You can follow the settings file to properly set everything up and then it should show up in the admin

**Comment by LuizDMM** [comment]
Hey @xtrinch, thank you for your answer!

I just checked the example project now. I thought that there was a "Notification" or "Message" model, and that it would be possible to send notifications from Django Admin by default. Ok, so it's possible to add this functionality to my app using device.send_message().

Thank you!

**Comment by xtrinch** [comment]
Not really, you can only send test messages to devices via the admin, not actual dynamic message objects. So yes, you'll have to call device.send_message programatically
